code/modules/projectiles/gun.dm

```diff
@@ -35,6 +35,7 @@
 	var/semicd = 0						//cooldown handler
 	var/weapon_weight = WEAPON_LIGHT
 	var/list/restricted_species
+	var/bolt_open = 0
```

Comment:
```suggestion
	var/bolt_open = FALSE
```

Reply:
@S34NW Thanks for the review!

I imagine people will have a lot of opinions about the loadout so I'm gonna wait for a bit before pushing my changes, but locally updated it like you said to remove the bloat :)